pallets/passkey/src/mock.rs

```diff
@@ -1,23 +1,24 @@
 //! Mocks for the Time-release module.
-
-use super::*;
+use common_primitives::{msa::MessageSourceId, node::AccountId};
 use frame_support::{
 	construct_runtime,
 	traits::{ConstU32, Everything},
 };
 use sp_core::H256;
-use sp_runtime::{traits::IdentityLookup, BuildStorage};
+use sp_runtime::{
+	traits::{Convert, ConvertInto, IdentityLookup},
+	AccountId32, BuildStorage,
+};
 
 use crate as pallet_passkey;
 
-pub type AccountId = u128;
 impl frame_system::Config for Test {
 	type RuntimeOrigin = RuntimeOrigin;
 	type RuntimeCall = RuntimeCall;
 	type Nonce = u64;
 	type Hash = H256;
 	type Hashing = ::sp_runtime::traits::BlakeTwo256;
-	type AccountId = AccountId;
+	type AccountId = MessageSourceId;
```

Comment:
AccountId is different from MsaId. These also should be a type that allows us to check signature in tests. In my PR I believe the types which are set allows this.

Reply:
True, got confused , add the correct AccountId

Reply:
I update mock on my side, Merging after your PR would be ideal